webrender/src/platform/macos/font.rs

```diff
@@ -92,9 +94,14 @@ impl FontContext {
     pub fn new() -> FontContext {
         debug!("Test for subpixel AA support: {}", supports_subpixel_aa());
 
+        // Force CG to use sRGB color space to gamma correct.
+        let contrast = 0.0;
+        let gamma = 0.0;
+
         FontContext {
             cg_fonts: HashMap::new(),
             ct_fonts: HashMap::new(),
+            gamma_lut: gamma_lut::GammaLut::new(contrast, gamma, gamma),
```

Comment:
given that you pass the same gamma as both paint gamma and device gamma, the whole LUT'ing thing becomes just a contrast application, according to the source:
```rust
if (src - dst).abs() < (1.0 / 256.0) {
        let mut ii : f32 = 0.0;
        for i in 0..256 {
            let raw_srca = ii / 255.0;
            let srca = apply_contrast(raw_srca, adjusted_contrast);

            table[i] = round_to_u8(255.0 * srca);
            ii += 1.0;
        }
}
```
Is this by design? Are we going to use different gamma parameters here in the long term?